utils/report.h

```diff
@@ -17,7 +17,10 @@ enum avg_mode {
 struct report_time_stat {
 	uint64_t sum;
 	uint64_t rec; /* time in recursive call */
+	uint64_t sum_sq;
+	uint64_t rec_sq;
 	uint64_t avg;
+	uint64_t std;
```

Comment:
It doesn't need to be an integer, why not using `double` type?

Reply:
Hi, that is a good point! I'm using the `uint64_t` in consistent with the argument type of the print_time function. 

(line 229 in tui.c) `static void print_time(uint64_t ntime)`

I think we do not need to break this consistency to STDV calculation, and it is sufficiently accurate to reflect the distribution. ^_^